adapter/internal/ga/client.go

```diff
@@ -77,6 +76,7 @@ type APIEvent struct {
 	RevisionUUID     string
 	IsDeployEvent    bool
 	OrganizationUUID string
+	DeployedEnv      string
```

Comment:
what is deployedEnv ?  is it a UUID or a name? If it is a name, can we guarantee that it is unique for a given org. (Since each project can have its own env, there is a possibility two projects can have the same env name)

Reply:
Here we are using the env name along with APIUUID and RevisionUUID. So I think APIUUD, RevisionUUID, DeployedEnv all together brings that uniqueness here.

Reply:
APIUUID is a unique one, and since the API belongs to exactly one project I think your argument stands.